antenna/ext/gcs/crashstorage.py

```diff
@@ -170,3 +184,33 @@ def save_crash(self, crash_report):
 
         self.save_dumps(crash_id, dumps)
         self.save_raw_crash(crash_id, raw_crash)
+
+    def load_crash(self, crash_id):
+        """Load crash data."""
+        raw_crash = {}
+        dumps = {}
+
+        raw_crash_key = self._get_raw_crash_path(crash_id)
+        try:
+            raw_crash = json.loads(self._load_file(raw_crash_key))
+        except Exception as exc:
```

Comment:
`load_file` will only be used in tests. I don't know what exceptions get raised by GCS library, so I left this general.

Reply:
it raises `google.cloud.exceptions.NotFound`

Reply:
Thank you!

Reply:
for consistency with s3, i think that `self._load_file` should catch `google.cloud.exceptions.NotFound` and raise `KeyNotFound`, and `KeyNotFound` should be caught here. If we then add `KeyNotFound` as a class attribute here and on `S3CrashStorage` it will be accessible for things that take either s3 or gcs.